src/tools/pkcs11-tool.c

```diff
@@ -147,8 +147,23 @@ static struct ec_curve_info {
 	{"secp256k1",		"1.3.132.0.10", "06052B8104000A", 256, 0},
 	{"secp521k1",		"1.3.132.0.35", "06052B81040023", 521, 0},
 
-	{"edwards25519","1.3.6.1.4.1159.15.1", "130c656477617264733235353139", 255, CKM_EC_EDWARDS_KEY_PAIR_GEN},
-	{"curve25519", "1.3.6.1.4.3029.1.5.1", "130b63757276653235353139", 255, CKM_EC_MONTGOMERY_KEY_PAIR_GEN},
+	/* Some of the following may not yet be supported by the OpenSC module, but may be other modules */
+	/* OpenPGP extensions by Yubikey and GNUK are not defined in RFCs, so pass by printable string */
+	/* See PKCS#11 3.0 2.3.7 */
+	{"edwards25519", "1.3.6.1.4.11591.15.1", "130c656477617264733235353139", 255, CKM_EC_EDWARDS_KEY_PAIR_GEN}, /* send by curve name */
+	{"curve25519",   "1.3.6.1.4.3029.1.5.1", "130a63757276653235353139", 255, CKM_EC_MONTGOMERY_KEY_PAIR_GEN}, /* send by curve name */
```

Comment:
I think these two also miss the `1` -- please, double-check.

Reply:
Fixing typo.